pandas/tests/io/test_sql.py

```diff
@@ -1425,14 +1422,6 @@ def test_database_uri_string(self, test_frame1):
         tm.assert_frame_equal(test_frame1, test_frame3)
         tm.assert_frame_equal(test_frame1, test_frame4)
 
-    @td.skip_if_installed("pg8000")
```

Comment:
Why did this test need to be removed?

Reply:
you are right. This doesn't belong here.